src/04-modality-specific-files/02-magnetoencephalography.md

```diff
@@ -40,7 +40,11 @@ limit.
 For example Neuromag/Elekta/Megin, which can produce several files
 for a single recording.
 Both `some_file.fif` and `some_file-1.fif` would belong to a single recording.
-In BIDS, the [`split`](../99-appendices/09-entities.md#split) entity is RECOMMENDED to deal with split files.
+In BIDS, the [`split`](../99-appendices/09-entities.md#split) entity is RECOMMENDED to deal with split files. 
+If there are split files and the optional `scans.tsv` is provided, remember to list all parts
+of the split file in `scans.tsv` and that the entries for the `acq_time` column in `scans.tsv`
+MUST be identical, as described in
```

Comment:
@eort I think you just missed this one in the recent set of changes.

Reply:
yes, I have. Thanks for pointing it out!

Reply:
Yes, I did. Thanks for pointing out!

Reply:
Sorry for the delay in responding, but it looks like you _reverted_ one of the text changes in eaf6e62 instead of updating the last text snippet, so now only 1 of the 3 duplicate text snippets reflect the new changes.

Reply:
Sorry, should be fixed now.

Reply:
It looks like you reverted the last snippet to the old text instead of using the new text in the other snippets.

Reply:
It actually became a hybrid version. Given your initial remark on that particular issue, the current version should also be fine (the phrase `split file` is not used anymore), but if you think it is better I can also change 
> all parts separately

to 

> all files separately

Reply:
Ah, I see. Thanks! I think saying "files" instead of "parts" is key, since "part" is ambiguous, while "file" isn't, so if you could change that, that would be great.